docs/en/operation/Admin-CLI.md

```diff
@@ -362,3 +396,158 @@ alluxio.master.audit.logging.enabled
+data-worker-1      DECOMMISSIONED   1                capacity      10.67GB          2.9.0            abcde
+                                                     used          0B (0%)
+data-worker-0      DECOMMISSIONED   2                capacity      10.67GB          2.9.0            abcde
+                                                     used          0B (0%)
+data-worker-2      ACTIVE           0                capacity      10.67GB          2.9.0            abcde
+                                                     used          0B (0%)
+
+# Then you can restart the decommissioned workers. The workers will start normally and join the cluster.
+$ ssh data-worker-0
+$ ./bin/alluxio-start.sh worker
+...
+
+# Now you will be able to observe those workers become ACTIVE again and have a higher version
+$ ./bin/alluxio fsadmin report capacity
+...
+Worker Name        State            Last Heartbeat   Storage       MEM              Version          Revision
+data-worker-1      ACTIVE           1                capacity      10.67GB          2.9.1            hijkl
+                                                     used          0B (0%)
+data-worker-0      ACTIVE           2                capacity      10.67GB          2.9.1            hijkl
+                                                     used          0B (0%)
+data-worker-2      ACTIVE           0                capacity      10.67GB          2.9.1            hijkl
+                                                     used          0B (0%)
+
+# You can run I/O tests against the upgraded workers to validate they are serving, before moving to upgrade the next batch
+$ bin/alluxio runTests --workers data-worker-0,data-worker-1
+```
+
+**Exit Codes**
+
+This command is idempotent and can be retried, but the admin is advised to manually check if there's an error. 
+
+The return codes have different meanings:
+
+1. `0(OK)`: All workers are successfully decommissioned and now idle. Safe to kill or restart this batch of workers now.
+2. `1(DECOMMISSION_FAILED)`: Failed to decommission all workers. The admin should double check the worker addresses and the primary master status.
+3. `2(LOST_MASTER_CONNECTION)`: Lost connection to the primary master while this command is running. This suggests the configured master address is wrong or the primary master failed over.
+4. `3(WORKERS_NOT_IDLE)`: Some workers were still not idle after the wait. Either the wait time is too short or those workers failed to mark decommissioned. The admin should manually intervene and check those workers.
+5. `10(LOST_SOME_WORKERS)`: Workers are decommissioned but some or all workers lost contact while this command is running. If a worker is not serving then it is safe to kill or restart. But the admin is advised to double check the status of those workers.
+
+### enableWorker
+
+The `enableWorker` command is the reverse operation of `decommissionWorker -d`.
```

Comment:
Explain in what circumstances such a command would be useful?

Reply:
added a few lines